cmd/burrow/commands/configure.go

```diff
@@ -203,6 +207,28 @@ func Configure(output Output) func(cmd *cli.Cmd) {
 				conf.GenesisDoc.ChainName = *chainNameOpt
 			}
 
+			if *restoreDumpOpt != "" {
+				if conf.GenesisDoc == nil {
+					output.Fatalf("no GenesisDoc/GenesisSpec provided, cannot restore dump")
+				}
+
+				if len(conf.GenesisDoc.Validators) == 0 {
+					output.Fatalf("On restore, validators must be provided in GenesisDoc or GenesisSpec")
+				}
+
+				state, err := execution.MakeGenesisState(db.NewMemDB(), conf.GenesisDoc)
+				if err != nil {
+					output.Fatalf("could not generate state from genesis: %v", err)
+				}
+
+				err = state.LoadDump(*restoreDumpOpt, conf.GenesisDoc)
```

Comment:
We are adding the GenesisDoc here... We write to out so I think that is fine - but is it what we want? See my other comment with LoadDump

Reply:
Removed